src/controllers/connection/helpers.ts

```diff
@@ -1,12 +1,13 @@
 import { InternalError } from '../../errors.js'
-import { ILogger } from '../../logger.js'
+import { BasicLogger } from '../../logger.js'
```

Comment:
Again I don't understand why the switch is being done here

Reply:
Mainly because when mocking `req.log` the `@Request req: express.Request` it did not like ILogger and <typeof pino> and was asking to kinda be more accurate with the types, to be fair have not tried with as Request, might just be fine